invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -75,6 +75,215 @@
     "document_pid": {
       "type": "string",
       "title": "Document ID"
+    },
+    "document_types": {
+      "items": {
+        "type": "string",
+        "enum": ["book", "proceedings", "report", "standard", "thesis"],
+        "default": "book"
+      },
+      "minItems": 1,
+      "title": "Document types",
+      "type": "array",
+      "uniqueItems": true
+    },
+    "language": {
+      "items": {
+        "enum": [
```

Comment:
Migration it is going to be fun here :)

Reply:
it was not that bad: https://github.com/CERNDocumentServer/cds-dojson/blob/97abcdcffed04e695d69d15f0c42117d255cc407/cds_dojson/marc21/fields/books/base.py#L480